docs/pages/api-reference/sink_connectors/s3.md

```diff
@@ -11,12 +11,12 @@ Data connector to sink data to S3.
 A name to identify the sink. The name should be unique across all Fennel connectors.
 </Expandable>
 
-<Expandable title="aws_access_key_id" type="Optional[str]" defaultVal="None">
+<Expandable title="aws_access_key_id" type="Optional[str] | Optional[Secret]" defaultVal="None">
```

Comment:
We do not support secrets for S3 sinks as of now. Can you add a note regarding that and change this field type to Optional[str]

Reply:
Done.